site/app/controllers/forum/ForumController.php

```diff
@@ -567,6 +575,12 @@ private function editThread(){
             if(!$this->checkThreadEditAccess($thread_id)) {
                 return false;
             }
+            if(empty($_POST['lock_thread_date'])){
```

Comment:
Besides checking if it is empty, I would suggest also checking if the date is a valid timestamp or not.